tests/plugins/org.polarsys.capella.test.model.ju/src/org/polarsys/capella/test/model/ju/menu/CapellaMenusTestCase.java

```diff
@@ -110,6 +114,18 @@ protected void testCopyAsTextCommand(EObject selectedElement) {
     } catch (ExecutionException e) {
     }
   }
+  
+  protected void testCopyAsTextNumericValue(Integer value) {
+    PrimitiveWrapper wrapper = new PrimitiveWrapper(value);
+    ExecutionEvent event = createExecutionEvent(wrapper);
+    String result = new CopyTextHandler() {
+      protected IStructuredSelection getSelection() {
+        IEvaluationContext context = (IEvaluationContext) event.getApplicationContext();
+        return (IStructuredSelection) context.getVariable("selection");
+      }
+    }.getSelectionAsText();
+    assertEquals("5\r\n", result);
```

Comment:
Not sure about \r\n on the running junit linux platform but we will see junits, 🤞